assets/src/components/d3/createResourceAccessChart.js

```diff
@@ -250,17 +251,17 @@ function createResourceAccessChart ({ data, weekRange, gradeSelection, resourceT
         : selection[0] - dy
 
     // Make sure the page doesnt scroll
-    d3.event.stopPropagation()
+    event.stopPropagation()
     // Move the brush
     gBrush.call(brush.move, [topSection, topSection + size])
   }
 
-  function brushmove () {
+  function brushmove (event) {
     const fullRange = mainYZoom.range()
-    const selection = d3.event
-      ? d3.event.selection[1] === 0 // prevents [0, 0] from being returned, which causes bug
-        ? [0, 0.1]
-        : d3.event.selection
+    const selection = event
+      ? event.selection[1] === 0 // prevents [0, 0] from being returned, which causes bug
+          ? [0, 0.1]
+          : event.selection
```

Comment:
Nit-pick: check whether indentation is correct.

Reply:
I think I indented as suggested, but I have a different linter so I am not sure if it same as your.

Reply:
Looks good.